lib/pci.py

```diff
@@ -19,9 +19,10 @@
 
 import re
 import os
-import commands
 import platform
 
+from .helper import runcmd
```

Comment:
Should we integrate pci and wrapper for now ?
The reason this was not done is to have it separated whenever needed.

Reply:
not getting you, you mean to move all methods in pci.py to helper.py?

Reply:
it is better to keep this way, as sooner or later we need this piece to be out of wrapper right....

Reply:
Yes, so should it use runcmd ?

Reply:
yep, till it resides here, its fine, and when we decide to move somewhere else, it can carry `runcmd` defintion aswell..